lib/taiko.js

```diff
@@ -585,6 +587,13 @@ module.exports.openTab = async (
   if (!/^https?:\/\//i.test(targetUrl) && !/^file/i.test(targetUrl)) {
```

Comment:
@zabil Can we fail fast here.

```
  if (!options.name) {
    throw new TypeError('Tab name needs to be provided');
  }
```

Reply:
I did not add this check because unlike `incognitoWindow` a tab identifier using `options.name` is optional parameter. The user can choose to pass the identifier example calls would be

```
openTab('google.com')
openTab('GitHub.com', {name: 'GitHub'}):
```

Both should be allowed.